src/apps/graph.ts

```diff
@@ -98,11 +103,16 @@ function main() {
       console.log();
 
       for (const flow of flows) {
-        if (
+        if (args.t) {
+          if (args.t === flow.node.key) {
+            console.log(graph.formatFlow(flow, options));
+            console.log();
+          }
+        } else if (
           args.f !== flow.node.spec.key &&
-          (args.r || flow.node.isEndpoint || args.t === flow.node.key)
+          (args.r || flow.node.isEndpoint)
```

Comment:
This would read much clearer if showRouters was used instead of args.r

Reply:
Perhaps the general pattern for this comment and the preceding one is to define aliases for all of the command line flags at the top of the function, e.g.

~~~
const showRouters = args.r;
const from = args.f;
~~~

Then the remaining code can reference the aliases instead of the flags. This also allows one to change the flags without changing the logic. I will see if I can slot this in to the rebasing I am doing tonight.

Reply:
I'll add the fix for these comments to PR #10

Reply:
Fix is in 6a7fa42999db41596c5c501e13a9c193c3486cf0